test/unit/reducer/test-reducer-utils.hpp

```diff
@@ -0,0 +1,100 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef __TEST_REDUCER_UTILS_HPP__
+#define __TEST_REDUCER_UTILS_HPP__
+
+#include "camp/resource.hpp"
+#include "gtest/gtest.h"
+
+#include "RAJA_unit_forone.hpp"
+
+//
+// Unroll types for gtest testing::Types
+//
+template <class T>
+struct Test;
```

Comment:
this should be deduplicated with the forall header at some point.

Reply:
Yes, we'll do this in a different PR.